meta/appveyor.yml

```diff
@@ -26,7 +26,7 @@ install:
   - rustc --version
   - cargo --version
   - rustup target add wasm32-unknown-unknown
-  - cargo install cargo-generate --features vendored-openssl
+  - cargo install --force --features vendored-openssl cargo-generate
```

Comment:
This is not the solution we want to have. `--force` reinstalls the binary, which takes about 10 minutes since everything is re-compiled to release code. We need a way to not install when it exists already in the cache.

Reply:
Okay.... and we have to detect that.

Reply:
Let's see if `cargo install` has some way to do that

Reply:
Odd... this shouldn't error:

```
 cargo install --features vendored-openssl cargo-generate && echo okay
    Updating crates.io index
     Ignored package `cargo-generate v0.5.0` is already installed, use --force to override
okay
```